bin/create-scripts/block.js

```diff
@@ -1,18 +1,24 @@
+#!/usr/bin/env node
+
 /** @format */
 const path = require( 'path' );
 const webpack = require( 'webpack' );
 
+// TODO: move this to the npm script, see #26258
+process.env[ 'SKIP_FLAG_IMAGES' ] = 'true';
```

Comment:
can we leave this for a separate PR? the flags while annoying aren't getting in our way at least since we don't have to include them in the sites themselves.

(it's good to get rid of them but I don't see the need to in this PR)

Reply:
Yep, I cherry-picked the commit from https://github.com/Automattic/wp-calypso/pull/26260 to make testing this PR easier for now. Once that's in, I'll rebase the commit away from this PR.